src/System.Net.Http/tests/FunctionalTests/RepeatedFlushContent.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class RepeatedFlushContent : StringContent
+    {
+        public RepeatedFlushContent(string content) : base(content)
+        {
+        }
+        
+        protected override Task SerializeToStreamAsync(Stream stream, TransportContext context)
+        {
+            stream.Flush();
+            stream.Flush();
```

Comment:
Yes, the repeated `Flush` is intentional. We had a previous bug about this when our WinHttpHandler stream implementation did not have idempotent `Flush`.  So, we added this test.